src/Symfony/Component/Process/Tests/ProcessTest.php

```diff
@@ -48,6 +48,43 @@ protected function tearDown()
         }
     }
 
+    /**
+     * @group legacy
+     * @expectedDeprecation The provided cwd does not exist. Command is currently ran against getcwd(). This behaviour is deprecated since version 3.4 and will be removed in 4.0.
+     */
+    public function testInvalidCwd()
+    {
+        if (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN') {
+            $this->markTestSkipped('Windows handles this automatically.');
+        }
+
+        // Check that it works fine if the CWD exists
+        $cmd = new Process('echo test', __DIR__);
+        $cmd->run();
+
+        $cmd = new Process('echo test', __DIR__.'/notfound/');
+        $cmd->run();
+    }
+
+    /**
+     * @group legacy
+     * @expectedDeprecation The provided cwd does not exist. Command is currently ran against getcwd(). This behaviour is deprecated since version 3.4 and will be removed in 4.0.
+     * @expectedException \PHPUnit_Framework_Error_Warning
```

Comment:
should be replaced by an `@expectedException`+`@expectedExceptionMessage` as suggested above

Reply:
Done